catalogs/catalog-jdbc-postgresql/src/main/java/org/apache/gravitino/catalog/postgresql/operation/PostgreSqlSchemaOperations.java

```diff
@@ -167,7 +171,11 @@ protected boolean isSystemDatabase(String dbName) {
   }
 
   private String getShowSchemaCommentSql(String schema) {
-    return String.format("SELECT obj_description('%s'::regnamespace) as comment", schema);
+    return String.format(
```

Comment:
why changing this?

Reply:
If the schema has an upper case word, the query will not work.